source/TextBlade.ConsoleRunner/NewGameRunner.cs

```diff
@@ -9,19 +9,20 @@ namespace TextBlade.ConsoleRunner;
 /// <summary>
 /// Terribly named; the thing that does all your new-game stuff.
 /// </summary>
-public class NewGameRunner
+public class NewGameRunner(Game game)
 {
-    private JObject _gameJson = null!;
-    private Game _game = null!;
-
-    public NewGameRunner(Game game)
+    private JObject? _gameJson;
+    private JObject GameJson
```

Comment:
Do we need this? `_gameJson` is only ever used in this class.

Reply:
This is a trick I use when I have a member that I want to use as a non-nullable but need to assign in a method that isn't a constructor. Since GameJson is actually assigned during ShowGameIntro(), this makes the NullReferenceexcxeption explicit if it isn't called or is called after another method that access the game's JSON object (I will clarify the exception text)